content/troubleshooting/common-issues.md

```diff
@@ -28,6 +27,35 @@ ssh://XXXXXXXX@git-codecommit.us-west-2.amazonaws.com/v1/repos/my_repo.git
 Please note that the ***XXXXXXXXX** refers to your **SSH-Key-ID**
 
 
+### Cannot access the repository. Request is unauthorized (401)
+
+###### Description
+
+When fetching or adding repositories from Github, Gitlab, Bitbucket, and others, you might encounter below error:
+
+```
+"Cannot access the repository. Request is unauthorized (401). Please check your credentials to access ..."
+```
+or
+```
+Repository is not accessible. Check access credentials and firewall settings...
+```
+This could happen due to a number of reasons such as:
+1. Repository settings were changed
+2. Access credentials are not valid e.g. provided SSH key is either expired or malformed or any other reason
+3. Repository is behind a firewall and requires IP addresses to be whitelisted
+4. OAuth access token should be refreshed
+
+
+###### Solution
+
+The following suggestions can help resolve the issue:
+
+1. Verify that the access credentials e.g. SSH key pairs were added correctly
+2. In general, ensure that the repository access is up to date. You can find more information [here](https://docs.codemagic.io/getting-started/adding-apps/#modifying-access)
+3. Confirm that the relevant IP addresses are [whitelisted](https://docs.codemagic.io/getting-started/adding-apps/#firewall-configuration-for-privately-hosted-repositories)
+4. Refresh the OAuth integration by going to Team settings, disconnecting, and then reconnecting the integration.
```

Comment:
The path is a bit unclear, could be Teams > Select your team > Team integrations > click to disconnect and re-connect